modules/efk-logging-elasticsearch-comp-status.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-elasticsearch.adoc
+
+[id="efk-logging-elasticsearch-comp-status-{context}"]
+= Viewing Elasticsearch status
+
+You can view the status of your Elasticsearch cluster.
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+. Change to the `openshift-logging` project.
++
+----
+$ oc project openshift-logging
+----
+
+. To view the Elasticsearch cluster status:
+
+.. Get the name of the Elasticsearch instance:
++
+----
+$ oc get Elasticsearch
+
+NAME            AGE
+elasticsearch   5h9m
+----
+
+.. Get the Elasticsearch status:
++
+----
+$ oc get Elasticsearch <Elasticsearch-instance> -o yaml
+----
++
+For example:
++
+----
+$ oc get Elasticsearch elasticsearch -n openshift-logging -o yaml
+----
++
+The output includes information similar to the following:
++
+----
+status: <1>
+  clusterHealth: green <2>
+  conditions: <3>
+      .....
+  nodes:  <4>
+      .....
+  pods: <5>
+      .....
+  shardAllocationEnabled: "True" <6>
```

Comment:
this has been updated to be similar to what Elasticsearch keeps internally. it would be `all` in this case (other possible value is `none` e.g. allocation is disabled)